forward_engineering/ddlProvider.js

```diff
@@ -101,10 +102,13 @@ module.exports = (baseProvider, options, app) => {
 			const deActivatedColumns = columns.filter(column => !column.isActivated).map(({ column }) => column);
 			const partitionsStatement = commentIfDeactivated(partitions, { isActivated: isPartitionActivated });
 
+			const [compositePK] = primaryKey
+			const compositePkFieldsNamesList = compositePK?.compositePrimaryKey.map(({name: columnName}) => columnName)
+			const compositePrimaryKeyOutlineConstraint = `PRIMARY KEY (${compositePkFieldsNamesList.join(', ')}) NOT ENFORCED`
 			const tableStatement = assignTemplates(templates.createTable, {
 				name: tableName,
 				column_definitions: externalTableOptions?.autodetect ? '' : '(\n' + tab(
-					[activatedColumns.join(',\n'), deActivatedColumns.join(',\n')].filter(Boolean).join('\n'),
+					[activatedColumns.join(',\n'), deActivatedColumns.join(',\n'), compositePrimaryKeyOutlineConstraint].filter(Boolean).join('\n'),
```

Comment:
Oh, that is unused code because that primaryKeyConstraint will be located in the position after closed bracket on table creation statement, so, it is redundant